ui/src/app/pages/za-settings/dashboard/dashlets/call-requests-dashlet/call-requests-dashlet.component.ts

```diff
@@ -384,5 +424,6 @@ export class CallRequestsDashletComponent implements OnInit, OnDestroy {
 
   ngOnDestroy(): void {
     this.destoyed.next(null);
+    this.destoyed.complete();
```

Comment:
for what?

Reply:
already resolved in upgradeClientBase.
Resolve in clientFilters branch too.